rsts/deployment/configuration/general.rst

```diff
@@ -374,14 +442,28 @@ which are applied when constructing a Pod. Typically, these options map one-to-o
 with K8s Pod fields. This makes it difficult to maintain configuration options as K8s
 versions change and fields are added/deprecated.
 
-*******************
-Example PodTemplate
-*******************
+*********************************
+Evaluation Order in PodTemplates
+*********************************
+
+The following diagram shows the precedence in evaluation order between the different types of PodTemplates and K8s Plugin Configuration.
+
+.. mermaid::
+   :alt: Evaluation order of PodTemplates
+
+   graph BT
+     B["@task pod_template"] --> A["k8s plugin"]
+     C["runtime PodTemplate"] --> B
+     D["@task pod_template_name"] --> B
+
 
-To better understand how Flyte constructs task execution Pods based on the default
-PodTemplate and K8s plugin configuration options, let's take an example. 
+To better understand how Flyte constructs task execution Pods based on Compile-time and Runtime PodTemplates,
+and K8s plugin configuration options, let's take a few examples.
 
-If you have the default PodTemplate defined in the ``flyte`` namespace
+Example 1: Runtime PodTemplate and K8s Plugin Configuration
```

Comment:
I think some of these examples may be redundant. Example 3 covers Example 1 and Example 2 right? OK to leave all as well.

Reply:
Yeah, I just want to make it clear that we can mix and match (up to a point).